<?xml version="1.0"?>
<launch>
  <node pkg="hls_lfcd_lds_driver" type="hlds_laser_publisher" name="hlds_laser_publisher" output="screen">
    <param name="port" type="string" value="/dev/ttyS1"/>
    <param name="frame_id" value="laser"/>
    <param name="robot_frame_id" value="base_link"/>
    <param name="laserGen" type="int" value="3"/>
    <param name="block_angle_1" type="int" value="9"/>
    <param name="block_angle_2" type="int" value="83"/>
    <param name="block_angle_3" type="int" value="128"/>
    <param name="block_angle_4" type="int" value="203"/>
    <param name="block_angle_5" type="int" value="248"/>
    <param name="block_angle_6" type="int" value="322"/>
    <param name="block_range" type="int" value="14"/>
    <param name="angle_min" type="double" value="-180"/>
    <param name="angle_max" type="double" value="180"/>
    <param name="delay_when_republish" type="int" value="0"/>
  </node>
</launch>

